ajax/libs/ng-jsoneditor/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "ng-jsoneditor",
+  "filename": "ng-jsoneditor.min.js",
+  "version": "1.5.1",
```

Comment:
I can't find v1.5.1, and I check the files you add are the same with v1.51.
So would you let the version number the same with upstream?
Using `1.51` is good here, because when we add old versions with auto-update config, the version number always the same withe upstream.

Thanks.